src/quantum/azext_quantum/_params.py

```diff
@@ -4,8 +4,20 @@
 # --------------------------------------------------------------------------------------------
 # pylint: disable=line-too-long
 
-from knack.arguments import CLIArgumentType
+import argparse
+from knack.arguments import CLIArgumentType, CLIError
 
+# pylint: disable=protected-access
+class JobParamsAction(argparse._AppendAction):
+    def __call__(self, parser, namespace, values, option_string):
+        params = {}
+        for item in values:
+            try:
+                key, value = item.split('=', 1)
+                params[key] = value
+            except ValueError:
+                raise CLIError('Usage error: {} KEY=VALUE [KEY=VALUE ...]'.format(option_string))
```

Comment:
This usage should be deprecated, but it's added for consistency with the rest of the extension. The following item tracks the pending work to replace it across the entire extension:
https://github.com/Azure/azure-cli-extensions/issues/3805